src/main/java/com/cloudbees/jenkins/plugins/customtools/CustomToolInstallWrapper.java

```diff
@@ -208,14 +211,15 @@ public Launcher decorateLauncher(AbstractBuild build, final Launcher launcher,
                     continue;
                 }
                 CustomToolsLogger.logMessage(listener, installed.getName(), "Label specifics from '"+spec.getLabel()+"' will be applied");
-                               
-                if (spec.hasAdditionalVars()) {
-                    additionalVarInjectors.add(EnvVariablesInjector.Create(spec.getAdditionalVars()));
+                    
+                final String additionalLabelSpecificVars = spec.getAdditionalVars();
+                if (additionalLabelSpecificVars != null) {
+                    additionalVarInjectors.add(EnvVariablesInjector.create(additionalLabelSpecificVars));
                 }
             }
             
             CustomToolsLogger.logMessage(listener, installed.getName(), "Tool is installed at "+ installed.getHome());
-            String homeDirVarName = (convertHomesToUppercase ? installed.getName().toUpperCase() : installed.getName()) +"_HOME";
+            String homeDirVarName = (convertHomesToUppercase ? installed.getName().toUpperCase(Locale.ENGLISH) : installed.getName()) +"_HOME";
```

Comment:
So you can;t support a tool installed when using a Turkish locale with Turkish filenames containing an `i` ?